Look into our eslint warnings

Can any of these be fixed?

```
/Users/humphd/repos/telescope/src/backend/utils/elastic.js
  116:10  warning  Prefer await to then()  promise/prefer-await-to-then

/Users/humphd/repos/telescope/src/backend/utils/wiki-feed-parser.js
  59:10  warning  Prefer await to then()  promise/prefer-await-to-then

/Users/humphd/repos/telescope/src/frontend/src/pages/index.js
  43:6  warning  React Hook useEffect has missing dependencies: 'numPages' and 'posts'. Either include them or remove the dependency array. You can also do a functional update 'setNumPages(n => ...)' if you only need 'numPages' in the 'setNumPages' call  react-hooks/exhaustive-deps
  60:8  warning  Prefer await to then()                                                                                                                                                                                                                        promise/prefer-await-to-then
  81:6  warning  React Hook useEffect has a missing dependency: 'getPostsCount'. Either include it or remove the dependency array                                                                                                                              react-hooks/exhaustive-deps
  92:6  warning  React Hook useEffect has a missing dependency: 'callback'. Either include it or remove the dependency array                                                                                                                                   react-hooks/exhaustive-deps

✖ 6 problems (0 errors, 6 warnings)
```

`master` has this when I build:

```
warn ESLintError:
/Users/humphd/repos/telescope/src/frontend/src/components/AboutFooter/AboutFooter.jsx
  97:15  warning  img elements must have an alt prop, either with meaningful text, or an empty string for
decorative images  jsx-a11y/alt-text

✖ 1 problem (0 errors, 1 warning)
warn ESLintError:
/Users/humphd/repos/telescope/src/frontend/src/components/MyFeedsPage/DeleteFeedDialogButton.jsx
  83:13  warning  The autoFocus prop should not be used, as it can reduce usability and accessibility for
users  jsx-a11y/no-autofocus

✖ 1 problem (0 errors, 1 warning)
warn ESLintError:
/Users/humphd/repos/telescope/src/frontend/src/pages/index.js
  85:6  warning  React Hook useEffect has missing dependencies: 'nextPageLink' and 'posts'. Either include
them or remove the dependency array. You can also do a functional update 'setPosts(p => ...)' if you only need
 'posts' in the 'setPosts' call  react-hooks/exhaustive-deps

✖ 1 problem (0 errors, 1 warning)
```

--- a/src/frontend/src/pages/index.js
+++ b/src/frontend/src/pages/index.js
@@ -83,6 +83,8 @@ export default function IndexPage() {
     }
 
     getPosts();
+    // Disabling the eslint check as nextPageLink and posts will cause the page to not render properly
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [telescopeUrl, numPages]);
 
   function getNewPosts() {

